x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/support-template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "index_patterns": ["content-chat-support-v1-*"],
+  "template": {
+    "mappings": {
+      "properties": {
+        "metadata": {
+          "properties": {
+            "support": {
+              "properties": {
+                "status": {
+                  "type": "keyword"
+                },
+                "assignee": {
+                  "properties": {
+                    "name": {
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
+                    },
+                    "email_address": {
+                      "type": "keyword"
+                    },
+                    "id": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "category": {
+                  "type": "keyword"
+                },
+                "customer": {
+                  "properties": {
+                    "name": {
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
+                    },
+                    "email_address": {
+                      "type": "keyword"
+                    },
+                    "id": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "resolved_at": {
+                  "type": "date"
+                },
```

Comment:
We might want to have fields to keep track of the status lifecycle / updates, wdyt? With one entry with each status change (e.g created, assigned, resolved, reopened and so on)? 

WDYT?

Reply:
Hmm that's a good point. I do worry that would make us assume too much about the system's lifecycle though. I mean, I can guess at a Salesforce lifecycle, but will that still hold true for Zendesk or Service Now? Maybe just a multi-value field like:
```
{
  "lifecycle_event": "keyword",
   "lifecycle_date": "date"
 }
 ```
 ?